src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -36,12 +36,12 @@ const defaultProps = {
 };
 
 const ReportActionItemCreated = (props) => {
-    const icons = ReportUtils.getIcons(props.report, props.personalDetails);
-
-    if (ReportUtils.isMoneyRequestReport(props.report.reportID) || ReportUtils.isTaskReport(props.report)) {
+    if (!ReportUtils.isChatReport(props.report)) {
```

Comment:
I think we might still want to check for moneyRequest report, because users can request money from a workspace chat, not just 1:1 chat reports.

Reply:
I'm not sure I see the point here. Workspace chats are of type `chat`. So the welcome message would still appear.

Reply:
@0xmiroslav This is regarding the 4th point mentioned in PR details
> The <ReportActionItemCreated /> (welcome message, avatars and bg art) will be rendered only if the report is a chat report.

We want to show the welcome message only if the report is a chat report.

| Before | After |
|--------|-------|
|   <img src="https://github-production-user-asset-6210df.s3.amazonaws.com/16493223/237837074-0a6431d2-2850-4745-870d-8754daa5e832.png" />     |  <img src="https://github-production-user-asset-6210df.s3.amazonaws.com/16493223/238158065-65f5088d-8bb1-4248-8786-44a7b2ebdc8a.png" />     |

Reply:
Hmm, I'm not sure I understand your question here.

If the report is a chat report (e.g. DM, Group, Room and Workspace chats) we will show the welcome message. Otherwise (IOU, expenseChat, Task) we won't.

Let me know if that answers your concern

Reply:
> what was not working before and what does this change fix?

https://github.com/Expensify/App/pull/18775#discussion_r1194287708

Reply:
Sorry for the confusion all. I think I simply missed the `!` logic inversion when reviewing 😅

Reply:
I agree with this: 

> So instead of: expense || iou || task. Let's just do !chat.

Reply:
Cool cool. Making this as resolved then :rocket: